src/screens/resources/podcasts-mobile.js

```diff
@@ -0,0 +1,89 @@
+import { podcasts } from "./resources";
+import { DotNavigation } from "../videos/dot-navigation";
+import { cx } from "../../utils/cx";
+import { videos } from "../videos/videos";
+import { useRef, useState } from "react";
+
+export function PodcastsMobile({ className }) {
+  const [visiblePodcastKey, setVisiblePodcastKey] = useState(videos[0].id);
+  const podcastsRef = useRef({});
+
+  const scrollTo = (podcastKey) => {
+    setVisiblePodcastKey(podcastKey);
+
+    podcastsRef.current[podcastKey].scrollIntoView({
+      behavior: "smooth",
+      block: "nearest",
+      inline: "center",
```

Comment:
Since these are re-used a few times maybe we could consider storing them in a central location to make the code shorter overall. 

We could also creating a util so we could just call `scrollRefIntoView(ref)` here

Reply:
You're right. I've added a function to the dot navigation component, since it always goes with it (we might want to scroll differently elsewhere).

Btw I took the opportunity to move that dot navigation component as well as the vertical scrollbar one to components/, where they should reside.